docs/channels/sms.rst

```diff
@@ -104,15 +104,15 @@ The following fields are available:
 
 **Language** - This allows you to set the language of this Text Message.
 
-**Published** - This allows you to set the published status of the Text Message. Unpublished Text Messages aren't sent.
+**Available** - This allows you to set the availability status of the Text Message. Unavailable Text Messages aren't sent.
 
 **Contact Segment** - This allows you to define the Segment/s who should receive the Text Message.
 
 .. vale off
 
-**Publish at (date/time)** - This allows you to define the date and time at which this Text Message is available for sending to Contacts
+**Enable at (date/time)** - This allows you to define the date and time at which this Text Message is available for sending to Contacts
```

Comment:
Likewise the SMS bundle doesn't yet have the option to enable/disable, maybe an upcoming feature?
![screenshot-mautic ddev site-2024 06 13-15_35_55](https://github.com/mautic/user-documentation/assets/2930593/4f9474a4-081f-4948-8ce5-c2a5aa8af8a4)